docs/higher-order/tca.ipynb

```diff
@@ -0,0 +1,1448 @@
+{
```

Comment:
<p><code>It can be seen in the components plot below that, for some factors, there is differential activity of the neurons between each brain region. The same can be said for the trial factors between the first block of stimulus trials and the second block.</code></p><p> </p><p>I suggest removing the interpretation here and add a paragraph "After" the plot. I will write something   below. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/307/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>